smartlearner/tasks/tests/test_stopping_criteria.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
```

Comment:
Not particularly. Let's put them at the root so when I started using PyCharm they won't clutter the autocomplete :P.